spec/support/unit/matchers/fail_with_message_matcher.rb

```diff
@@ -20,15 +20,17 @@ def supports_block_expectations?
       end
 
       def failure_message
-        msg = "Expectation should have failed with message '#{expected}'"
+        lines = ["Expectation should have failed with message:"]
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.